scripts/download_pix3d.py

```diff
@@ -21,15 +23,17 @@
     if not os.path.exists(pix3d_dir):
         os.makedirs(pix3d_dir)
 
-    # download the zip file, which contains all the obj files
+    # download the zip file, which contains all the obj files. Size ~3.5 GB
     print("Download the zip file, may take a while:")
     pix3d_url = "http://pix3d.csail.mit.edu/data/pix3d.zip"
     zip_file_path = os.path.join(pix3d_dir, "pix3d.zip")
-    urlretrieve(pix3d_url, zip_file_path)
+    urlretrieve(pix3d_url, zip_file_path, ProgressBar())
 
     # unzip the zip file
     print("Unzip the zip file.")
     cmd = "unzip {}".format(zip_file_path)
     subprocess.call(cmd, shell=True, cwd=os.path.dirname(zip_file_path))
 
-    os.remove(zip_file_path)
\ No newline at end of file
+    os.remove(zip_file_path)
+    os.remove(os.path.join(pix3d_dir, "img"))
```

Comment:
Have you tried this? 

`os.remove` only works on files not folders.

Reply:
Sorry, I didn't tried it initially but now it should work!